windows/make.vbs

```diff
@@ -1,2 +0,0 @@
-Set oShell = WScript.CreateObject("WSCript.shell")
-oShell.run "cmd /K title Building the installation package... |inno-compiler\ISCC.exe installer-source\KolibriSetupScript.iss & exit"
```

Comment:
Since this file is now empty, should it be deleted? @mrpau-julius

Reply:
@benjaoming yes it must be deleted.